src/zope/testing/renormalizing.txt

```diff
@@ -234,3 +234,59 @@ Combining a checker with something else does not work:
         ...
     TypeError: unsupported operand type(s) for +: ...
 
+Using the 2to3 exception normalization:
+
+    >>> from zope.testing.renormalizing import EXCEPTION_2TO3
+    >>> checker = OutputChecker()
+    >>> want = """\
+    ... Traceback (most recent call last):
+    ... foo.bar.FooBarError: requires at least one argument."""
+    >>> got = """\
+    ... Traceback (most recent call last):
+    ... FooBarError: requires at least one argument."""
+    >>> result = checker.check_output(want, got, EXCEPTION_2TO3)
+    >>> import sys
+    >>> if sys.version < '3':
+    ...     expected = True
+    ... else:
+    ...     expected = False
+    >>> result == expected
+    True
```

Comment:
We're not sure we understand what you mean.

We took the conscious decision to use Python 3 as the "baseline" so to say. Meaning, when running the tests in Python 3 we do not touch the output. This means, when fixing tests, you "move forward" so to say instead of sticking to Python 2.